eventstore/models.py

```diff
@@ -1094,3 +1094,39 @@ def clean(self):
                 date(self.dob_year, self.dob_month, self.dob_day)
             except ValueError as e:
                 raise ValidationError(f"Invalid date of birth date, {str(e)}")
+
+
+class OpenHIMQueue(models.Model):
+    class ObjectType:
+        PREBIRTH_REGISTRATION = "PrebirthRegistration"
+        CHW_REGISTRATION = "CHWRegistration"
+        PUBLIC_REGISTRATION = "PublicRegistration"
+        CHANNEL_SWITCH = "ChannelSwitch"
+        OPTOUT = "OptOut"
+        choices = (
+            (PREBIRTH_REGISTRATION, "PrebirthRegistration"),
+            (CHW_REGISTRATION, "CHWRegistration"),
+            (PUBLIC_REGISTRATION, "PublicRegistration"),
+            (CHANNEL_SWITCH, "ChannelSwitch"),
+            (OPTOUT, "OptOut"),
+        )
+
+    class Status:
+        PENDING = 0
+        PROCESSING = 1
+        COMPLETE = 2
+        ERROR = 3
+
+        choices = (
+            (PENDING, "Pending"),
+            (PROCESSING, "Processing"),
+            (COMPLETE, "Complete"),
+            (ERROR, "Error"),
+        )
```

Comment:
Are we on a new enough version of django that we can use `models.IntegerChoices` instead? https://docs.djangoproject.com/en/4.2/ref/models/fields/#enumeration-types

Reply:
I'll check